File: Modelo/Usuario.cs
namespace punto_de_venta_C_.Modelo
{
    public class Usuario
    {
        private int id;
        private string username;
        private string password;
        private string rol;
        private string nombre;
        private int edad;
        private string sexo;

        public Usuario(int id, string username, string password, string rol, string nombre, int edad, string sexo)
        {
            this.id = id;
            this.username = username;
            this.password = password;
            this.rol = rol;
            this.nombre = nombre;
            this.edad = edad;
            this.sexo = sexo;
        }

        public int getId() { return id; }
        public void setId(int id) { this.id = id; }
        public string getUsername() { return username; }
        public void setUsername(string username) { this.username = username; }
        public string getPassword() { return password; }
        public void setPassword(string password) { this.password = password; }
        public string getRol() { return rol; }
        public void setRol(string rol) { this.rol = rol; }
        public string getNombre() { return nombre; }
        public void setNombre(string nombre) { this.nombre = nombre; }
        public int getEdad() { return edad; }
        public void setEdad(int edad) { this.edad = edad; }
        public string getSexo() { return sexo; }
        public void setSexo(string sexo) { this.sexo = sexo; }

        public bool esAdmin() { return "ADMINISTRADOR".Equals(this.rol); }

        public override string ToString()
        {
            return nombre + " [" + rol + "]";
        }
    }
}

File: Modelo/Usuario.py
class Usuario:
    def __init__(self, id=0, username='', password='', rol='', nombre='', edad=0, sexo=''):
        self._id = id
        self._username = username
        self._password = password
        self._rol = rol
        self._nombre = nombre
        self._edad = edad
        self._sexo = sexo

    def getId(self):
        return self._id

    def setId(self, id):
        self._id = id

    def getUsername(self):
        return self._username

    def setUsername(self, username):
        self._username = username

    def getPassword(self):
        return self._password

    def setPassword(self, password):
        self._password = password

    def getRol(self):
        return self._rol

    def setRol(self, rol):
        self._rol = rol

    def getNombre(self):
        return self._nombre

    def setNombre(self, nombre):
        self._nombre = nombre

    def getEdad(self):
        return self._edad

    def setEdad(self, edad):
        self._edad = edad

    def getSexo(self):
        return self._sexo

    def setSexo(self, sexo):
        self._sexo = sexo

    def esAdmin(self):
        return self._rol == "ADMINISTRADOR"

    def __str__(self):
        return f"{self._nombre} [{self._rol}]"